client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -926,6 +931,20 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
         return `/agenda/${this.motion.agenda_item_id}/speakers`;
     }
 
+    /**
+     * Click handler for the pdf button
+     */
+    public onDownloadPdf(): void {
+        // this.pdfService.open(docDefinition);
+
+        const doc = this.motionPdfService.motionToDocDef(this.motion, this.lnMode, this.crMode);
+        console.log(JSON.stringify(doc));
+
+        // open oor download
+        this.pdfDocumentService.open(doc);
```

Comment:
move this to service.

Reply:
Make a motion-pdf-export service. This holds all the stuff for configuring the export (via dialog), and holding all export things together.